frontend/tests/tests.test.js

```diff
@@ -34,19 +34,27 @@ for (const envName in deploymentConfigs) {
 
   // todo: rename this as I add onto it. I think I want to do the workflow from search -> comparison
   test(envName + ': ' + 'Cset search - select, load, compare', async ({ page }) => {
+    const testCodesetId = '1000002363';
     // Load page and click menu
     await page.goto(appUrl);
+
+    // close alert panel if it appears
+    const alertPanelClose = await page.waitForSelector('[data-testid=flexcontainer-Alerts] button');
```

Comment:
Does it correctly skip past this if not testing in local? I can check too.

Reply:
oh, i don't know

Reply:
Yep, it was failing if not on local. Fixed!